### DISCUSSION TITLE
Keep same counter despite extension change between saving?

### DISCUSSION THREAD

**Discussion by audioscavenger** [audioscavenger/save-image-extended-comfyui#9] [Polls]
Currently changing extension starts a new counter

**Comment by aaronnewsome** [comment]
There should also be a way to turn off counter digits entirely

**Reply by audioscavenger** [reply]
this is done since v2.80

**Comment by audioscavenger** [comment]
Discussion closed, the counter does NOT increment when saving same image with no changes, but simply swapping file extensions.
